server/pages/index.js

```diff
@@ -530,6 +530,7 @@ function setUpCSP( req, res, next ) {
 			'https://widgets.wp.com',
 			'*.wordpress.com',
 			'https://apis.google.com',
+			'https://appleid.cdn-apple.com',
```

Comment:
This fixes the following error:

```
[Report Only] Refused to load the script 'https://appleid.cdn-apple.com/appleauth/static/jsapi/appleid/1/en_US/appleid.auth.js' because it violates the following Content Security Policy directive: "script-src 'self' 'report-sample' 'unsafe-eval' stats.wp.com https://widgets.wp.com *.wordpress.com https://apis.google.com 'nonce-aecbe1c2fef4a48e9a61ae5660b530ba01e69aafd140e9b8c2203c6b72ff0191b9d8ced5f3dfb111edae5ca66eefcab2' www.google-analytics.com 'sha256-3yiQswl88knA3EhjrG5tj5gmV6EUdLYFvn2dygc0xUQ=' 'sha256-ZKTuGaoyrLu2lwYpcyzib+xE4/2mCN8PKv31uXS3Eg4='". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```